pkg/app/web/src/components/header/index.tsx

```diff
@@ -206,11 +206,23 @@ export const Header: FC = memo(function Header() {
             Events
           </Link>
         </MenuItem>
-        <MenuItem>
+        <MenuItem divider={true}>
           <Link component={RouterLink} to={PAGE_PATH_SETTINGS}>
             Settings
           </Link>
         </MenuItem>
+        <MenuItem>
+          <Link
+            href="https://pipecd.dev/docs/"
+            target="_blank"
+            rel="noreferrer"
+          >
+            Docs
+          </Link>
+        </MenuItem>
+        <MenuItem disabled={true} dense={true}>
+          {process.env.STABLE_VERSION?.substring(0, 10)}
```

Comment:
```suggestion
          Version: {process.env.STABLE_VERSION?.substring(0, 10)}
```
nits

Reply:
btw, not sure why do we need to cut the string, why not just show the full version string? Since the last part is the commit hash and can be used to refer to source code on pipecd repo, which could be useful in some cases 👀

Reply:
<img width="388" alt="image" src="https://user-images.githubusercontent.com/49914427/156151439-ff588a6c-1d97-4313-92b6-2d1b74da48d2.png">
I think this one is easier to understand, but the display looks like this. Which one do you think is better?

Reply:
I get your point 👍 Then, how about remove the `Version: ` part and show the full version string?

Reply:
<img width="380" alt="image" src="https://user-images.githubusercontent.com/49914427/156152348-e0f84dd2-573e-48de-901d-6bb3dfbfa531.png">
Sorry, my comment was not updated and I missed it.
This is what it looks like when I put them all up. Personally, I find it too long.

Reply:
Normally, dirty is not displayed.

Reply:
I guess, making the `Docs` text longer (such as `Documentations`) will make the last line less long 😄 
We can try to shift all the text to the right side as well. Happy to hear other opinions cc @nghialv @knanao

Reply:
I vote for "Documentation" to be unified with tab name on our homepage.

Reply:
And it should open a new tab when clicking on the documentation link. So maybe we also need a new tab icon for it.

Reply:
The current implementation already has `target=_blank` properties for the `Docs` link, so I think it should be fine without the icon, I think only one option in a list that has a different icon is a bit odd.

Reply:
Right! I am fine without icon as well.

Reply:
> I vote for "Documentation" to be unified with tab name on our homepage.
And it should open a new tab when clicking on the documentation link. So maybe we also need a new tab icon for it.

I agree with this.
And it may be better to put an icon next to "Documentations" like this.
![Screen Shot 2022-03-01 at 20 18 05](https://user-images.githubusercontent.com/50069775/156160182-11b6527c-bb59-4070-b796-96cc4a408750.png)

Reply:
That is exactly what I was thinking about.

Reply:
About the version string, I think showing the full one is fine. Because on the production it is not that long.

Reply:
Looks so beautiful!

Just a nit:

How do you think about moving the menu popup to be lower to avoid overlapping the tab bar?
We can resolve it in another PR.